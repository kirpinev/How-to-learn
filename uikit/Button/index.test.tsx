import * as renderer from 'react-test-renderer'
import { Button } from './index'

describe('Index page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Button primary>Buy</Button>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
