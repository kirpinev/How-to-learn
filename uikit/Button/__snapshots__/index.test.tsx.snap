// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
<button
  className="Button-nk2hqu-0 hxSCrT"
>
  Buy
</button>
`;
